aws-datastore/src/androidTest/java/com/amplifyframework/datastore/MultiAuthSyncEngineInstrumentationTest.java

```diff
@@ -807,12 +810,18 @@ private void configure(Class<? extends Model> modelType,
 
         // Setup DataStore
         DataStoreConfiguration dsConfig = DataStoreConfiguration.builder()
-                                                                .errorHandler(exception -> {
-                                                                    Log.e(tag,
-                                                                          "DataStore error handler received an error.",
-                                                                          exception);
-                                                                })
-                                                                .build();
+                                                .errorHandler(exception -> {
+                                                    Log.e(tag,
+                                                          "DataStore error handler received an error.",
+                                                          exception);
+                                                })
```

Comment:
we don't need `{}` for single liners :)
```
.errorHandler(error -> Log.e(tag, message, error))
```

but even better, wouldn't a Hub publish be more useful here for testing?

Reply:
Updated it to be a lambda expression. I only added this in here to make sure we could pick up on any exceptions thrown during the test. We can explore some alternatives as part of a different PR though.